lib/prometheus_exporter/instrumentation/unicorn.rb

```diff
@@ -51,13 +51,13 @@ def collect_unicorn_stats(metric)
 
     def worker_process_count
       return nil unless File.exist?(@pid_file)
-      pid = File.read(@pid_file)
+      pid = File.read(@pid_file).to_i
```

Comment:
To trim `\n` came from the pid file.

Without it, the method triggers like `pgrep -P 1\n -f unicorn -a` and it causes an errror: `sh: 2: -f: not found`.